packages/jest-jasmine2/src/jasmine/jasmine_light.js

```diff
@@ -88,6 +88,10 @@ exports.interface = function(jasmine: Jasmine, env: any) {
       return env.xit.apply(env, arguments);
     },
 
+    tit(description: string) {
```

Comment:
lol

Reply:
haha this was my first attempt at getting it to work - I thought I removed it 🤣